docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc

```diff
@@ -243,24 +244,27 @@ After getting a cup of coffee, you can run this binary directly:
 
 To test the application, open your browser and access the following URL:
 
-
 * http://localhost:8080/tokens[http://localhost:8080/tokens]
 
 If everything is working as expected, you are redirected to the Keycloak server to authenticate.
 
-To authenticate to the application, type the following credentials when at the Keycloak login page:
+To authenticate to the application, enter the following credentials at the Keycloak login page:
 
 * Username: *alice*
 * Password: *alice*
 
 After clicking the `Login` button, you are redirected back to the application and a session cookie will be created.
 
-The session for this demo is short-lived and you will be asked to re-authenticate on every page refresh. Please follow the Keycloak https://www.keycloak.org/docs/latest/server_admin/#_timeouts[session timeout] documentation to learn how to increase the session timeouts. For example, you can access Keycloak Admin console directly from Dev UI by selecting a `Keycloak Admin` link if you use xref:security-oidc-code-flow-authentication.adoc#integration-testing-keycloak-devservices[Dev Services for Keycloak] in dev mode:
+The session for this demo is valid for a short period of time and, on every page refresh, you will be asked to re-authenticate.
```

Comment:
```suggestion
The session for this demo is valid for a short period, and on every page refresh, you will be asked to re-authenticate.
```

Reply:
All the above suggestions make sense IMHO but this one looks like an over optimization :-), `period of time` reads clearer to me

Reply:
Thank you both for your suggestions. I will keep this one as is then, that is, keep as "period of time"